cms/models/permissionmodels.py

```diff
@@ -4,7 +4,11 @@
 from django.contrib.auth.models import Group, UserManager
 from django.contrib.sites.models import Site
 from django.core.exceptions import ImproperlyConfigured
-from django.utils import importlib
+try:
```

Comment:
If the import is moved after lif DJANGO_1_6 there is no need to use `importlib` because we will brake python2.6. For dajngo 1.6 there is no problem to use `from django.utils import importlib`